action/Scanning.action

```diff
@@ -0,0 +1,33 @@
+# Goal definition of a single scanning action.
+
+# Pre-defined constants to code the type of scanning action.
+int8 UNKNOWN_TYPE=0  # default value does not code a valid command
+int8 SHELF=1  # detect and localize shelf floors
+int8 BAR_CODE=2  # detect, recognize, and localize bar codes
+int8 SEPARATOR=3  # detect and localize separators
+int8 OBJECT_RECOGNITION=4  # detect, recognize, and localize front object in facing
+int8 OBJECT_COUNTING=5  # count all objects within one facing
+
```

Comment:
This definition does not allow us to do a full store inventory with one command. And if we want split it up and do it in the most efficient way, we'd need some kind of high level outside of the action server. Is this intended?